plugins/dell-kestrel/fu-dell-kestrel-ec.c

```diff
@@ -374,6 +374,9 @@ fu_dell_kestrel_ec_dock_data_cmd(FuDellKestrelEc *self, GError **error)
 	FuDellKestrelEcHidCmd cmd = FU_DELL_KESTREL_EC_HID_CMD_GET_DOCK_DATA;
 	g_autoptr(GByteArray) res = fu_struct_dell_kestrel_dock_data_new();
 
+	/* add delay to improve stability */
+	fu_device_sleep(FU_DEVICE(self), 500);
+
```

Comment:
can you talk more about this?  Is it just after a specific command it takes longer?  Because this is going to slow down _all_ commands.

Reply:
thanks, turned this delay into retry intervals.